runtime-modules/content/src/lib.rs

```diff
@@ -1850,6 +1850,8 @@ decl_module! {
 
             // Ensure top bid exists
             let top_bid = english_auction.ensure_top_bid_exists::<T>()?;
+            let top_bidder_id = top_bid.bidder_id;
+            let top_bidder_account_id = T::MemberAuthenticator::controller_account_id(top_bidder_id)?;
```

Comment:
So this is really a call which cannot fail unless the runtime is broken, but I also dont think we should have a panicable `ok` call either.

So why dont you do a `debug_assert(::is_ok)` call on the result, and add a big comment that this should never fail, since membership has been authenticated as real prior, and they are never deleted.

Reply:
added here: 7336a43021ccd917c153094a13d48535f1cca3b7